NEWAPI.md

```diff
@@ -32,124 +31,222 @@ know how to use `git` appropriately.
+Now we can build the package and run the unit tests:
 
-Run `./build.sh Your.ApiName.Here` to check it builds and the
-generated unit tests pass.
+```sh
+./build.sh Google.Cloud.Dialogflow.Cx.V3
+```
+
+If anything fails here, it's likely to be either an API
+configuration problem, or a generator problem. Either way, raise a
+GitHub issue - unless you're feeling particularly patient, it's
+probably worth waiting for someone on the team to look at it.
 
-Step 7: Commit just the changes for your API
---------------------------------------------
+If all has gone well, however (which it will do 90+% of the time),
+you can now commit everything:
 
-Create a commit containing the new directory under `apis`. If you
-ran `generateapis.sh` without specifying the API ID, and other APIs
-were regenerated with changes, don't include those changes in the
-commit.
+```sh
+git add --all
+git commit -m "Generate Google.Cloud.Dialogflow.Cx.V3"
 
+Step 6: Add smoke tests (where suitable)
+----------------------------------------
 
-Step 8: Create a PR
--------------------
+It's useful to be able to check that the service is actually alive
+and listening for requests. Release manager is able to detect good
+candidates for non-destructive smoke tests. Run it with the
+`suggest-smoke-tests` command, specifying the package ID:
 
-You should now have a lot of new project/coverage/doc/solution files, and
-your modified API catalog.
+```sh
+./prepare-release.sh suggest-smoke-tests Google.Cloud.Dialogflow.Cx.V3
+```
 
-Commit all of these, push and create a pull request. This should
-consist of two commits: one for the original codegen, and one for
-the API catalog and project files.
+For some APIs, there are no suitable (detected) methods. If that's
+the case, you'll see output ending in:
+
+```text
+Number of smoke tests suggested: 0
+```
 
-(You *can* do all of this in a single commit, but we've typically
-found that separating them helps diagnose issues with one part or
-another.)
+That's okay - it's not ideal, but it's not a blocker.
 
-Step 9: Merge the PR
---------------------
+If smoke tests *are* suggested, the output will end with a block of
+JSON and then a line saying that the JSON has been written to
+a `smoketests.json` file (within the `apis/your-package-id`
+directory).
 
-As everything is generated other than the API catalog change, the PR
-can be merged on green. Of course, a second pair of eyes on the
-change is always useful.
+**Please review smoke tests before running or committing them.**
+In particular, if the RPC sounds like it might be "dangerous" based
+on the name (such as the Vision API's `PurgeProducts` method),
+you'll want to look really, really carefully.
 
-Step 10 (Optional): Release the first package for the API
+Once you've reviewed the smoke tests, enable the API in the
+[Cloud Console API
+dashboard](https://console.cloud.google.com/apis/dashboard) and then
+run them with release manager. To run the smoke tests, use the
```

Comment:
nit: Maybe add a note here about default application credentials?

Reply:
Yup, have added just a line saying it assumes they're set up.